README.md

```diff
@@ -21,6 +21,37 @@
 
 Native Rust implementation of [Apache Iceberg](https://iceberg.apache.org/).
 
+## RoadMap
+
+### Catalog
+
+| Catalog Type | Status      |
+|--------------|-------------|
+| Rest         | Done        |
+| Hive         | In Progress |
+| Sql          | Not Started |
+| Glue         | Not Started |
+| DynamoDB     | Not Started |
+
+### FileIO
+
+| FileIO Type | Status      |
+|-------------|-------------|
+| S3          | Done        |
+| Local File  | Done        |
+| GCS         | Not Started |
+| HDFS        | Not Started |
+
+### Table API
+
+* Table reader, including planning, reading into arrow record batch, etc.
+* Table writer, including data writer, equality deletion writer, position deletion writer, etc.
```

Comment:
How about separating the Table API as the following so that we can list the feature status of each subpart. 🤔

### Table API

#### Metadata 
| Operation | Status      |
|-------------|-------------|
| GetManifest | Not support |
| GetManifestList | Not support |

#### Reader
...

#### Writer
| Writer | Status      |
|-------------|-------------|
| Data File Writer | Not Support |

Reply:
I think `GetManifest` maybe too detailed, by other things like planning should be included.